tools/mixxxdb_cleanup.sql

```diff
@@ -79,6 +79,10 @@ DELETE FROM track_analysis WHERE track_id NOT IN (SELECT id FROM track_locations
 -- Post-cleanup maintenance                                          --
 -----------------------------------------------------------------------
 
-VACUUM;
+-- Update statistics for the query planner
+-- https://www.sqlite.org/lang_analyze.html
+ANALYZE;
```

Comment:
What is the rational for Analyze before Vacuum? I would swap it intuitively. 
Can you add a comment?

Reply:
I don't know which order is correct and why to choose one over the other:

https://stackoverflow.com/questions/58178062/in-what-order-should-i-call-analyze-and-vacuum-in-an-sqlite3-database

Reply:
Found it:

https://sqlite.org/forum/forumpost/62fb63a29c5f7810?t=h

Richard Hipp himself recommends run VACCUM before ANALYZE.